### ISSUE TITLE
Cant create a new table

### ISSUE THREAD

**Issue by erenhan1** [jdglaser/fast-to-sql#12]
Hi,
I'm quite new in python and I'm trying to create a small ETL process, so in order to do that Im trying to upload df to sql server, my code was working very fine in local laptop until we move to company's server instance and I started to having error, basically my code was like that and it was working.
Working code:

```
import pandas as pd
import pyodbc
from fast_to_sql import fast_to_sql as fts
df=pd.read_excel(r'C:\Users\han.37\Desktop\Contact Pattern\Archive\Skill List.xlsx')
sql_conn = (
r'DRIVER={SQL Server Native Client 11.0};'
r'SERVER=GRATHLT37320;'
r'DATABASE=Performance;'
r'Trusted_Connection=yes;'

)
conn_db = pyodbc.connect(sql_conn)
create_statement = fts.fast_to_sql(df, 'cpattern' , conn_db, if_exists='replace', temp=False)

conn_db.commit()
conn_db.close()
```
Error Code:

```
import pandas as pd
import pyodbc
from fast_to_sql import fast_to_sql as fts
df=pd.read_excel(r'C:\Users\han.37\Desktop\Contact Pattern\Archive\Skill List.xlsx')
sql_conn = (
r'DRIVER={SQL Server Native Client 11.0};'
r'SERVER=GRATHDB03\REPLICA;'
r'DATABASE=Performance;'
r'Trusted_Connection=yes;'

)
conn_db = pyodbc.connect(sql_conn)
create_statement = fts.fast_to_sql(df, 'cpattern' , conn_db, if_exists='replace', temp=False)

conn_db.commit()
conn_db.close()
```
And the error is;
ProgrammingError: ('42000', '[42000] [Microsoft][SQL Server Native Client 11.0][SQL Server]The specified schema name "None" either does not exist or you do not have permission to use it. (2760) (SQLExecDirectW)')

But the issue is there is no permission, I have all the permissions by our IT team.

**Comment by jdglaser** [comment]
I will need to take a closer look at this later this week. Looks like it may have something to do with the `Server\Instance` Syntax for the driver connection.

**Comment by erenhan1** [comment]
> I will need to take a closer look at this later this week. Looks like it may have something to do with the `Server\Instance` Syntax for the driver connection.

Definitely, I succesfully sent the df with sqlalchemy but fast to sql was the way better option in terms of data upload speed, I look forward to hear something from you.

**Comment by jdglaser** [comment]
I've attempted to fix this issue with the latest release [2.1.12](https://github.com/jdglaser/fast-to-sql/releases/tag/v2.1.12). Please try again and let me know if it fixes the problem.

**Comment by erenhan1** [comment]
> I've attempted to fix this issue with the latest release [2.1.12](https://github.com/jdglaser/fast-to-sql/releases/tag/v2.1.12). Please try again and let me know if it fixes the problem.

it's working very fine and also I would like to really thank you for this great library, it's the fastest one I've ever used.

**Comment by jdglaser** [comment]
No worries, thanks for pointing out this issue! Glad you like it!
